.github/workflows/cicd.yaml

```diff
@@ -15,35 +15,20 @@ jobs:
     uses: ./.github/workflows/pr.yaml
     
   Build_Push_Images:
-    runs-on: ubuntu-latest
+    runs-on: self-hosted
     needs: [quality-check]
-    permissions:
-      contents: read
-      packages: write
+    permissions: write-all
     steps:
       - name: Checkout
         uses: actions/checkout@v2
 
-      - name: Free up Disk Space
-        run: |
-          # Remove specific Docker images that are no longer needed
-          docker rmi $(docker images -f "dangling=true" -q) || true
-          
-          # Prune unused Docker objects (images, containers, volumes, and networks)
-          docker system prune -f || true
-
-          # Clean up specific directories known to consume a lot of space
-          sudo rm -rf /path/to/large/directory
-
-          # Show disk space usage
-          df -h
-
       - name: Login to GHCR
         uses: docker/login-action@v1 
         with:
           registry: ghcr.io
           username: ${{ github.actor }}
-          password: ${{ secrets.GITHUB_TOKEN }}
+          password: ${{ secrets.GH_PAT }}
```

Comment:
hmm is it possible to skip custom PAT and use GITHUB_TOKEN itself?

Reply:
No. It used to be GITHUB_TOKEN here. I changed it back to GH_PAT because of freaking Bazel. It caches the credentials, but the GITHUB_TOKEN has a very short lifespan.